packages/kwordle/kwordle/js/sdk.js

```
@@ -0,0 +1,105 @@
+  });
+}
+
+function getDirectory(location) {
+  return new Promise(function (callback) {
+    // Get source of directory listing
+    fetchFile(location).then(function (data) {
+      // Create list
+      var files = [];
+
+      // Define regex for extracting different things
+      const tagReg = new RegExp('<a(\n|.)*?(?=<\/a>)', "gim");
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)